client/src/components/MainCarousel.jsx

```diff
@@ -3,57 +3,75 @@ import Button from 'react-bootstrap/Button';
 
 function MainCarousel() {
   return (
-    <Carousel data-bs-theme="dark" interval="3000" wrap="true" pause="false">
+    <Carousel data-bs-theme="light" interval="4000" wrap="true" pause="false">
       <Carousel.Item>
-        <img className="d-flex w-100" src=".\src\assets\golf_1.webp" alt="First slide" />
-        <Carousel.Caption>
-          <h5 style={{ fontFamily: "'Yeseva One', cursive" }}>First slide label</h5>
-          <p style={{ fontFamily: "'Josefin Sans', sans-serif", fontWeight: 400 }}>
-            Nulla vitae elit libero, a pharetra augue mollis interdum.
+        <img className="d-flex w-100" src=".\src\assets\golf_5.webp" alt="door club" />
+        <Carousel.Caption style={{ top: '40vh' }}>
+          <h5
```

Comment:
Inkább css-ben kéne egységesen megváltoztatni a font-family-t itt egy link hogy lehet egyszerűen: https://youtu.be/x_u-njAPbWc?si=dHYkMQi0VE4Gwdh5